api/app/signals/apps/services/domain/auto_create_children/actions.py

```diff
@@ -136,7 +136,8 @@ def run(self, signal):
             # Copy attachments to the child signal
             attachments_qs = signal.attachments.filter(is_image=True)
             if attachments_qs.count():
-                Signal.actions.copy_attachments(data=attachments_qs.all(), signal=child_signal)
+                # created_by=None, because this runs in a Celery task and no request.user is known here
+                Signal.actions.copy_attachments(data=attachments_qs.all(), signal=child_signal, created_by=None)
```

Comment:
With the change in the `extra_properties` the FE recently made these actions do not work anymore. They should be either refactored or deleted. By default these actions are turned off on the Amsterdam PROD environment.

Reply:
Will be removed/refactored in a new PR